jest.config.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  collectCoverage: true,
+  collectCoverageFrom: [
+    './src/**/*.[jt]s?(x)'
```

Comment:
We have only js file here, also we might never have have scenario of `.jsx` or `.tsx' file here